### PR TITLE
Update files to adhere to new lint requirements

### PR DESCRIPTION
**Pull request by umohnani8**
Signed-off-by: Urvashi Mohnani <umohnani@redhat.com>

### CODE REVIEW

**Comment by umohnani8** [comment]
@rhatdan @lsm5 @jwhonce PTAL

Will cut 4.4.0 after this gets in.

**Review by lsm5 — APPROVED** [review]
Linter is happy so I'm happy :smiley:  . But I'll still defer to @rhatdan or @jwhonce (in case he still wants to mess around here :smile: )

/lgtm
/hold

File: .cirrus.yml
Comment (umohnani8) [inline .cirrus.yml:22]: @cevich this is where I need to update the image right?

**Comment by jwhonce** [comment]
Python changes LGTM but Cirrus is not happy

File: .cirrus.yml
Comment (cevich) [reply .cirrus.yml:22]: correct.

**Comment by cevich** [comment]
@umohnani8 could you be more specific please, all I see is green check-marks.

**Comment by cevich** [comment]
Oh is it all the "Build Parsing Results" code annotations?  You can ignore those.

**Comment by umohnani8** [comment]
got it, thanks!

@lsm5 @jwhonce @rhatdan this is ready, can I please get a lgtm

**Pull request merged**
